ui/src/dev/server/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM node:alpine
 # Java for Swagger client generation
 RUN apk --no-cache add openjdk8-jre
 
-RUN mkdir /.config &&  mkdir /.npm && chmod a+rwx /.config /.npm
+RUN mkdir -p /.cache/yarn /.yarn                        \
+    && echo '--install.no-lockfile true' > /.yarnrc     \
```

Comment:
Can this configuration setting be in just one place?

Reply:
Mmm... as in `ADD` a file from the source code? I don't think I can do that with the way the Dockerfile is currently situated in a child directory of the UI project root - if it were in `workbench/ui` yeah, that'd be easy.  At any rate if we're enabling the lockfile then this line just becomes `touch /.yarnrc` - `yarn` gets fussy if it can't access the config file, whether there's anything in it or not.  I think if I don't explicitly create it and give it wide open permissions `yarn` doesn't have the ability to make a file there.

I could probably put a comment about that, otherwise it looks odd touching an empty config file.

Reply:
Love the comment, thank you. I just meant that since you have the setting in `ui/.yarnrc`, I would expect yarn to pull it from there so you would not also need the setting in the root `/.yarnrc`.